lib/impl/AwsS3Tvm.js

```diff
@@ -109,9 +111,47 @@ class AwsS3Tvm extends Tvm {
       secretAccessKey: stsRes.Credentials.SecretAccessKey,
       sessionToken: stsRes.Credentials.SessionToken,
       expiration: stsRes.Credentials.Expiration,
-      params: { Bucket: params.s3Bucket }
+      params: { Bucket: bucketName }
     }
   }
 }
 
+function checkAndCreateS3Bucket(namespace, prefix, region) {
+  return new Promise((resolve, reject) => {
+    var s3 = new AWS.S3({region: region});
```

Comment:
styling (trivial) change `var` to `const or let` and no `;` at the end of the line (run `npm run beautify`)